core/src/tag_utils.rs

```diff
@@ -151,11 +151,59 @@ impl SwfMovie {
         Self::from_data(&data, url.into(), loader_url)
     }
 
-    /// Construct a movie based on the contents of the SWF datastream.
+    /// Construct a movie based on the contents of a datastream.
     pub fn from_data(
         swf_data: &[u8],
```

Comment:
Just a nit: This name might no longer be the best fit. Nominating renaming it to just `data`.